js/index.js

```diff
@@ -40,3 +40,66 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+var hEdit = document.querySelectorAll('a');
```

Comment:
how can you refactor it here so your code is more DRY?

Reply:
I assume this could be done by using for loops to make all of the assignments considering that all of the assignments only differ based on the index that they are applied on. I pushed up some code that does this for the colors, but I still need to look into how to do this for textContent assignments because I would need to call the index on each assignement.